test/schema_test.ts

```diff
@@ -233,7 +234,32 @@ describe('Schema', () => {
       const normalizedBoo = (normalized as any).properties.Boo;
       // Make sure the function is preserved in spite of the missing value hint across
       // the deep copy that makeObjectSchema() does.
-      assert.deepEqual(normalizedBoo.items.autocomplete(), ['1']);
+
+      const options = normalizedBoo.items.options;
+      assert.isFunction(options);
+      assert.deepEqual(options({} as PropertyOptionsExecutionContext), ['1']);
+    });
+
+    it('options functions are disallowed in dynamic sync tables', () => {
```

Comment:
Added a test for `throwOnDynamicSchemaWithJsOptionsFunction` while I was in here